action.yml

```diff
@@ -1,31 +1,35 @@
-name: 'Run on Yarn'
-description: 'Composite GitHub Action to run a command with Yarn.'
+name: "Run on Yarn"
+description: "Composite GitHub Action to run a command with Yarn."
 inputs:
   command:
-    description: 'Command to run'
+    description: "Command to run"
     required: true
   node-version:
-    description: 'Node Version'
+    description: "Node Version"
     default: 14
-    required: false    
+    required: false
   github-token:
-    description: 'Github token to use when checking out the repo'
+    description: "Github token to use when checking out the repo"
     default: ${{ github.token }}
-    required: false        
+    required: false
+  fetch-depth:
+    description: "Checkout depth"
+    default: 1
 runs:
   using: "composite"
   steps:
     - uses: actions/checkout@v2
       with:
         token: ${{ inputs.github-token }}
+        fetch-depth: ${{ inputs.fetch-depth }}
```

Comment:
Hmm, this is now two props that are directly drilled to `actions/checkout`. What do you think about having an option like `checkout` / `checkout_options` / something like that?  Is such a thing possible?

Reply:
it would probably be possible to skip checkout with a varialbe and an `if` check? i assume that would work in these composite actions... not sure

Reply:
Yeah now that we have reusable workflows...